src/pages/components/tile/usage.mdx

```diff
@@ -457,7 +498,7 @@ hierarchical importance like navigation or catalog tiles.
 <Row>
 <Column colLg={12}>
 
-![Example image of how tiles are built on the narrow grid.](images/tile-usage-14.png)
+![Example image of how tiles are built on the narrow grid.](images/tile-usage-16.png)
```

Comment:
There are some typos in this image.

<img width="995" alt="Screen Shot 2023-02-14 at 9 10 30 AM" src="https://user-images.githubusercontent.com/43969356/218778552-02cede05-9531-4dda-be1d-0f027d7820cc.png">